content/docs/fairing/gcp-kubeflow-notebook.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Train and Deploy on GCP from a Kubeflow Notebook"
+description = "Use Kubeflow Fairing to train and deploy a model on Google Cloud Platform (GCP) from a notebook that is hosted on Kubeflow."
```

Comment:
In (most of) the Kubeflow docs, we've standardised on not having a full stop at the end of the description, since the description is often a phrase rather than a full sentence. (Plus, the lack of a full stop looks better, since the description appears a little like a subtitle and thus matches the title formatting better without the full stop.)

Reply:
Thanks for pointing that out.